src/Components/FilterGraduates/ByEnglishLevel.jsx

```diff
@@ -0,0 +1,52 @@
+import React, { Component } from "react";
+import { Dropdown } from "semantic-ui-react";
+
+class FilterByEnglishLevel extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      optionsEnglishLevel: [
+        {
+          key: "All",
+          text: "Todos",
+          value: "All",
+          label: { color: "red", empty: true, circular: true },
+        },
+        {
+          key: "Basic",
+          text: "Basico",
+          value: "Basico",
+          label: { color: "blue", empty: true, circular: true },
+        },
+        {
+            key: "Intermediate",
+            text: "Intermedio",
+            value: "Intermedio",
+            label: { color: "green", empty: true, circular: true },
+        },
+        {
+            key: "Advanced",
+            text: "Avanzado",
+            value: "Avanzado",
+            label: { color: "yellow", empty: true, circular: true },
+        }
+      ]
+    }
+  }
+
+  render () {
+    return  (
+      <Dropdown 
```

Comment:
I found a bug with this part, when you click on an item within the list the filter menu is still open. :( Maybe you guys could try something like <Dropdown.Menu> or <Dropdown.Item> :D